src/pages/HomePage/components/PopularDiets/PopularDiets.tsx

```diff
@@ -1,97 +1,101 @@
-                     <button className="kcal">1300 kcal</button>
-                     <button className="kcal">1500 kcal</button>
-                     <button className="kcal">2000 kcal</button>
-                </div>
-                <p className="taque">Taque earum rerum hic tenetur a sapiente maiores alias consequatur aut perferendis doloribus asperiores...</p>
-                <div className="frombutton">
-                 <div className="fromday">
-                     <p className="from">from</p>
-                     <p className="num">62</p>
-                     <p className="pln">pln</p>
-                     <p className="day"> /day</p>
-                 </div>
-                 <button className="choose">Choose</button>
-                </div>
-             </div>
-        </div>
+export const PopularDiets = () => {
+  const sequences = [
+    {
+      id: 1,
+      imgfood: Maskgroupslim,
+      text1: 'Slim',
+      kalori1: '1000 kcal',
+      kalori2: '1300 kcal',
+      kalori3: '1500 kcal',
+      text2:
+        'Taque earum rerum hic tenetur a sapiente maiores alias consequatur aut perferendis doloribus asperiores...',
+      fromtext: 'from',
+      numbertext: 57,
+      plntext: 'pln',
+      daytext: '/day',
+      choose: 'Choose',
+    },
+    {
+      id: 2,
+      imgfood: Maskgroupfit,
+      text1: 'Fit',
+      kalori1: '1800 kcal',
+      kalori2: '2000 kcal',
+      kalori3: '2500 kcal',
+      text2: 'Taque earum rerum hic tenetur a sapiente maiores...',
+      fromtext: 'from',
+      numbertext: 65,
+      plntext: 'pln',
+      daytext: '/day',
+      choose: 'Choose',
+    },
+    {
+      id: 2,
+      imgfood: Maskgroupvegan,
+      text1: 'Veqan',
+      kalori1: '1300 kcal',
+      kalori2: '1500 kcal',
+      kalori3: '2000 kcal',
+      text2: 'Taque earum rerum hic tenetur a sapiente maiores alias consequatur...',
+      fromtext: 'from',
+      numbertext: 59,
+      plntext: 'pln',
+      daytext: '/day',
+      choose: 'Choose',
+    },
+    {
+      id: 4,
+      imgfood: Maskgroupketo,
+      text1: 'Keto',
+      kalori1: '1300 kcal',
+      kalori2: '1500 kcal',
+      kalori3: '2000 kcal',
+      text2:
+        'Taque earum rerum hic tenetur a sapiente maiores alias consequatur aut perferendis doloribus asperiores...',
+      fromtext: 'from',
+      numbertext: 57,
+      plntext: 'pln',
+      daytext: '/day',
+      choose: 'Choose',
+    },
+  ];
+  return (
+    <div className="popular-diets narrow-container">
+      <p className="themost">The most popular diets</p>
+      <div className="arrows">
+        <button className="arrow1">
+          <img alt="errorimage" className="picarrow1" src={arrowleft} />
+        </button>
+        <button className="arrow2">
+          <img alt="errorimage" className="picarrow2" src={arrowright} />
+        </button>
+      </div>
+      <div className="fourdiets">
+        {sequences.map((sequence, index) => (
+          <PopularDiet
```

Comment:
you must add the "key" attribute here. please do it in the next task
(https://reactjs.org/docs/lists-and-keys.html)